src/index.js

```diff
@@ -3,12 +3,14 @@ import dotenv from "dotenv";
 import cors from "cors";
 dotenv.config();
 
-import usersRoutes from "./routes/users.js";
+import { authRoutes } from "./routes/auth.routes.js";
+import { userRouter } from "./routes/user.routes.js";
```

Comment:
Maybe we could use a more standardized pattern here: "authRoutes" and "userRouter"